plugin.php

```diff
@@ -12,12 +13,13 @@ function get_secure_menu_entries($page, $user)
         {
             $ret['Ticket System Data Entry']=$page->secure_root.'tickets/_admin/data.php';
         }
+        
         return $ret;
     }
 
     function get_plugin_entry_point()
     {
-        return array('name'=>'Ticket Registration/Transfer', 'link'=>'tickets/index.php');
```

Comment:
because i disliked the language flow of the menu
we don't register a ticket in the same way that we register an art car or a theme... 
and slashes in menu links are like contractions in titles..  best avoided..